jcheck/src/main/java/org/openjdk/skara/jcheck/IssuesTitleCheck.java

```diff
@@ -88,11 +88,6 @@ private boolean hasTrailingPeriod(String description) {
     }
 
     private boolean hasLeadingLowerCaseLetter(String description) {
-        if (Character.isUpperCase(description.charAt(0))) {
-            return false;
-        }
-        var firstWord = description.split(" ")[0];
-        // If first word contains special character, it's very likely a reference to file or function, ignore it
-        return !FILE_OR_FUNCTION_PATTERN.matcher(firstWord).matches();
+        return ALL_LOWER_CASE_PATTERN.matcher(description.split(" ")[0]).matches();
```

Comment:
I presume that `description` has been `trim`med? If so, then this should be fine without additional checks.

Reply:
Yes, the description has been trimmed